projects/scion/workbench-application-platform/src/lib/spec/qualifier.spec.ts

```diff
@@ -153,4 +162,79 @@ describe('Qualifier', () => {
       expect(matchesIntentQualifier(null, {entity: 'person'})).toBeFalsy();
     });
   });
+
+  describe('function \'patchQualifier(...)\'', () => {
+
+    it('patches intent qualifier with capability qualifier containing no wildcards', () => {
```

Comment:
Please rephrase so it is easier to understand what we exactly are testing here.

Reply:
Done